public/app/features/scopes/internal/ScopesDashboardsScene.tsx

```diff
@@ -1,50 +1,101 @@
-import { css } from '@emotion/css';
+import { css, cx } from '@emotion/css';
+import { isEqual } from 'lodash';
 import { Link } from 'react-router-dom';
 
-import { GrafanaTheme2, Scope, urlUtil } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { GrafanaTheme2, urlUtil } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectRef, SceneObjectState } from '@grafana/scenes';
 import { Button, CustomScrollbar, FilterInput, LoadingPlaceholder, useStyles2 } from '@grafana/ui';
 import { useQueryParams } from 'app/core/hooks/useQueryParams';
 import { t, Trans } from 'app/core/internationalization';
 
+import { ScopesFiltersScene } from './ScopesFiltersScene';
 import { fetchSuggestedDashboards } from './api';
+import { DASHBOARDS_OPENED_KEY } from './const';
 import { SuggestedDashboard } from './types';
+import { getScopeNamesFromSelectedScopes } from './utils';
 
 export interface ScopesDashboardsSceneState extends SceneObjectState {
+  filters: SceneObjectRef<ScopesFiltersScene> | null;
   dashboards: SuggestedDashboard[];
   filteredDashboards: SuggestedDashboard[];
+  forScopeNames: string[];
   isLoading: boolean;
+  isPanelOpened: boolean;
+  isEnabled: boolean;
   scopesSelected: boolean;
   searchQuery: string;
 }
 
+export const getBaseDashboardsState: () => Omit<ScopesDashboardsSceneState, 'filters'> = () => ({
+  dashboards: [],
+  filteredDashboards: [],
+  forScopeNames: [],
```

Comment:
Don't understand this name, what is this used for?

Reply:
`getInitialDashboardsState` should be more descriptive.

It's there in order to get the initial state of the Scene. It's extracted in a separate method as it's also used to reset the scene's state in various scenarios.

It's also a method as opposed to being a static object to due the value reading from local storage.